google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,162 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.gapic.devtools.clouddebugger.v2
```

Comment:
Well the proto package needs to match the directory structure, so the minimum change would be to move the API from `google/devtools/clouddebugger/` to `google/devtools/debugger/`. That would give us the API name we want for it.

Then if we also want it out of `/devtools/`, or into `/cloud/`, sure; but that's not a problem really: `google.cloud.gapic.{api}.{version}` is the same no matter the subdirectory.